modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -132,15 +132,15 @@ During the _Process_ phase, Mule starts pulling records from the stepping queue
 
 image::batch-diagram.jpg[batch+diagram]
 
-Note that a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. However, if you configure an aggregator, the batch job step changes its behavior while processing records, depending on how the aggregator is configured:
+Note that by default, a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. However, you can configure the batch job or the block size of the aggregator to change the behavior of the batch job step:
```

Comment:
I think "you can configure the batch job or the block size of the aggregator" is not correct because you are always configuring an aggregator and not only the batch job. Maybe just a rephrase of the original text will be better?